docs/src/input.md

```diff
@@ -594,15 +584,13 @@ Shortcuts such as `"Control+o"` or `"Control+Shift+T"` are supported as well. Wh
 Note that you still need to specify the capital `A` in `Shift-A` to produce the capital character. `Shift-a` produces a lower-case one as if you had the `CapsLock` toggled.
 
 
```

Comment:
Replace "This method focuses the selected element and produces a single keystroke." with "The [`method: Locator.press`] method focuses the selected element and produces a single keystroke."